assets/javascript/script.js

```diff
@@ -54,7 +56,7 @@ function mobileMenu() {
   const appearMobileLink3 = document.createElement('a');
   appearMobileLink3.classList.add('mobile-link');
   appearMobileUlLi3.appendChild(appearMobileLink3);
-  appearMobileLink3.setAttribute('href', '#Contact');
```

Comment:
Thank you for your friendly and professional review! 

I've fixed the issue with broken links by changing the first capital letters in the `href` attribute of an `<a>` tags.